### PR TITLE
Added debian control files. Fixed ticket 8.

### PR DESCRIPTION
**Pull request by grzetzp**
Files dpcs and dpcs-settings were created because there was a need single executable files without .py extension.

Private modules should be defined in debian/install file. - Compiled form. Proposed destination: /usr/share/dpcs

### CODE REVIEW

**Comment by bdfhjk** [comment]
I've reviewed control files today and they look fine. We need to set up an installation process for the package to be operative, but it's out of the scope of this ticket.

Good job :+1:

**Comment by bdfhjk** [comment]
@MSusik  Could you take a look before I'll merge?

File: debian/rules [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/make -f
+
+%:
+	dh $@ --with python2
```
Comment (MSusik) [inline debian/rules:4]: Better way would be to use `setup.py` from Python's `distutils`. Then we would be able to have the most current versions of the dependencies.

**Comment by bdfhjk** [comment]
I've used this code and have created a fixed debian control files. Thanks!

**Pull request closed without merging**